app/flatpak-builtins-build-finish.c

```diff
@@ -237,6 +237,7 @@ collect_exports (GFile          *base,
     "share/gnome-shell/search-providers", /* Search providers */
     "share/appdata",                      /* Copy appdata/metainfo files (legacy path) */
     "share/metainfo",                     /* Copy appdata/metainfo files */
+    "share/man/man1",                     /* Manpages */
```

Comment:
It doesn't seem clear that section 1 man pages are the only man pages flatpaks might want to export. For example, flatpak-builder is distributed as a flatpak and has a section 5 man page, `flatpak-manifest`. For reference, from `man man`:

```
1.         Executable programs or shell commands
2.         System calls (functions provided by the kernel)
3.         Library calls (functions within program libraries)
4.         Special files (usually found in /dev)
5.         File formats and conventions eg /etc/passwd
6.         Games
7.         Miscellaneous (including macro packages and conventions), e.g. man(7), groff(7)
8.         System administration commands (usually only for root)
9.         Kernel routines [Non standard]
```